.github/guides/STANDARDS.md

```diff
@@ -97,7 +97,7 @@ While we normally encourage (and in some cases, even require) bringing out of da
 
 * Files and path accessed and referenced by code above simply being #included should be strictly lowercase to avoid issues on filesystems where case matters.
 
-### Signal Handlers
+### RegisterSignal() - Signal Handlers
```

Comment:
?

Reply:
Just figured I'd do it since they're so close to each other

Reply:
If you're gonna make like a category, why not do 
```
### RegisterSignal()

#### Signal Handlers

#### Signal Override
```
Or is that just my compulsion kicking in

Reply:
Sure